modules/nodes-nodes-garbage-collection-containers.adoc

```diff
@@ -15,6 +15,24 @@ When eviction thresholds are set for garbage collection, the node tries to keep
 
 * *eviction-hard* - A hard eviction threshold has no grace period, and if observed, {product-title} takes immediate action.
 
+The following table lists the eviction thresholds:
+
+.Variables for configuring container garbage collection
+|===
+| Node condition | Eviction signal | Description
+
+| MemoryPressure 
+| memory.available 
+| The available memory on the node.
+
+| DiskPressure 
+a| * nodefs.available
+  * nodefs.inodesFree 
+  * imagefs.available
+  * imagefs.inodesFree
+| The available disk space or inodes on the node root file system (`nodefs`) or image file system (`imagefs`).
```

Comment:
I'd probably offset these with commas instead of enclosing them in parentheses because the spacing looks a little weird.